packages/cli/src/models/files/index.ts

```diff
@@ -1,9 +1,10 @@
-import ZosVersion from './ZosVersion';
+import { ZOS_VERSION, checkVersion } from './ZosVersion';
```

Comment:
This is a more elegant fix to what I did in https://github.com/zeppelinos/zos/pull/525 to fix this problem. So when it comes to merge that PR, we should resolve the conflict favoring this solution.